Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1597,33 +1596,59 @@ def add_member_to_group(default_base_dn):
     search_base = args.get('base-dn') or default_base_dn
 
     # get the  dn of the member - either user or computer
-    args_err = "Please provide either username or computer-name"
+    args_err = "Please provide either username, computer-name, or nested_group_cn"
     member_dn = ''
 
     if args.get('username') and args.get('computer-name'):
         # both arguments passed
         raise Exception(args_err)
     if args.get('username'):
-        member_dn = user_dn(args['username'], search_base)
+        usernames = argToList(args.get('username'))
+        demisto.debug(json.dumps(usernames, indent=2))
```

Comment:
```suggestion
        demisto.debug(f"Usernames collected are {usernames}")
```

Reply:
This line is observed in the last version and is not part of updates in this PR.